src/flyff/flyffentity.js

```diff
@@ -32,6 +32,7 @@ export default class Entity {
     };
     skillLevels = {};
     activeBuffs = {};
+    activePartyBuffs = {};
```

Comment:
This needs to be handled in `serialize` and `unserialize` similar to the other objects on entities